addons/sale_timesheet/views/project_task_views.xml

```diff
@@ -220,4 +220,33 @@
                 </xpath>
             </field>
         </record>
+
+        <record id="project_milestone_view_form" model="ir.ui.view">
+            <field name="name">project.milestone.view.form.inherit</field>
+            <field name="model">project.milestone</field>
+            <field name="inherit_id" ref="project.project_milestone_view_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="//group[@name='main_details']" position="after">
+                    <group t-if="record.allow_billable.raw_value">
+                        <field name="project_partner_id" invisible="1"/>
+                        <field name="sale_order_line"/>
+                        <field name="quantity"/>
```

Comment:
I think we could use the `percentage` widget, but I think this field must be a float because the widget do `value * 100`  before displaying the value into the UI (to check). 😇 
```suggestion
                        <field name="quantity" widget="percentage"/>
```